### PR TITLE
Preserve comments in rewrite namespace exports pass.

### PR DESCRIPTION
**Pull request by rictic**
Analyzing and linting polymer 3 sources was having lots of issues due to load-bearing-comments being missing.

### CODE REVIEW

File: src/passes/rewrite-namespace-exports.ts [hunk 1]
```diff
@@ -121,9 +121,12 @@ class RewriteNamespaceExportsPass {
         correctedNamespaceName = 'Polymer';
         name = 'Polymer';
       }
-      nodePath.replace(jsc.exportNamedDeclaration(jsc.variableDeclaration(
-          this.mutableNames.has(correctedNamespaceName) ? 'let' : 'const',
-          [jsc.variableDeclarator(jsc.identifier(name), exportedExpression)])));
+      replacePreservingComments(
```
Comment (FredKSchott) [inline src/passes/rewrite-namespace-exports.ts]: so many lines! Can you split these up a bit with a few variable declarations?

File: src/test/unit/project-converter_test.ts [hunk 2]
```diff
@@ -910,10 +910,14 @@ export function increment() {
       });
       assertSources(await convert(), {
         'test.js': `
+/**
```
Comment (FredKSchott) [inline src/test/unit/project-converter_test.ts]: if comment is empty like this, we should continue to remove it

File: src/test/unit/project-converter_test.ts [hunk 3]
```diff
@@ -2951,6 +2963,52 @@ console.log('second script');
...
+             */
+            Polymer.MyMixinFunction = function() {};
+          </script>
+        `
+      });
+
+      assertSources(await convert(), {
+        'test.js': `
+/**
+             * This is a long and important comment.
```
Comment (FredKSchott) [inline src/test/unit/project-converter_test.ts]: can we fix this formatting?

File: src/test/unit/project-converter_test.ts (hunk 2 above)
Comment (rictic) [reply src/test/unit/project-converter_test.ts]: Seems reasonable.

File: src/test/unit/project-converter_test.ts (hunk 3 above)
Comment (rictic) [reply src/test/unit/project-converter_test.ts]: The recast API is not well documented. I poked around in the debugger but didn't find anything that did exactly what we want. Hacked something together that seems to do all right though.

**Comment by rictic** [comment]
Comments addressed, PTAL

**Comment by rictic** [comment]
Thanks!

**Pull request merged**